src/resolvers/projectResolver.ts

```diff
@@ -333,21 +333,38 @@ export class ProjectResolver {
     searchTerm?: string,
   ) {
     if (!searchTerm) return query;
+    const SIMILARITY_THRESHOLD = 0.4;
```

Comment:
While having a default value for threshold, please allow overriding it by an entry in configuration.